### ISSUE TITLE
retries

### ISSUE THREAD

**Issue by wlodi83** [mesos/chronos#385]
Hi,

how can I update the retries section and set it to 0 value?

I have:

"retries": 2

but I would like to have 0 - and 0 by default.

**Comment by elingg** [comment]
Setting it to 0 in your job definition should work

**Comment by msiebeneicher** [comment]
+1

**Comment by msiebeneicher** [comment]
The retries option is not available in the chronos ui (v 2.3.3-1.0.1.debian77). Is the api currently the only possibility to change this handling?

**Comment by elingg** [comment]
Through the REST API currently, although this could be useful to add to the Web UI.

**Comment by msiebeneicher** [comment]
Yes, it would be useful. It would be also nice to change the default value by the global chronos configuration. In our case we can use the Web UI now only for an overview .
